test/visualData.ts

```diff
@@ -74,12 +72,13 @@ export class VisualData extends TestDataViewBuilder {
                     },
                     values: this.valuesBottomValues
                 }
-            ], columnNames).build();
+            ], <string[]>columnNames).build();
     }
 }
 
 export function getRandomHexColor(): string {
```

Comment:
lets replace this function with: 

```
export function getRandomHexColor(): string {
    const hex: string = Math.floor(Math.random() * 16777215).toString(16); //Multiplying by 16777215 (which is 0xFFFFFF in hexadecimal) ensures that the random number covers the full range of possible hex colors.
    return `#${hex.padStart(6, '0')}`;
}
```

And remove `getHexColorFromNumber()` and `getRandomInteger()` functions. 
So, it will be clearer and easier to maintain further.